### PR TITLE
Completes questions

### PR DESCRIPTION
**Pull request by hollisjamison**

### CODE REVIEW

File: HIGHLIGHTS.md [hunk 1]
```diff
@@ -7,3 +7,14 @@ What highlights can you come up with to share with the class?
 What do you really like? What can you find for fun facts related to any of these topics online?
 
 ## Cohort Highlights
+I'm very proud of my portfolio template (Although I haven't added the actual data yet)
+
+https://hollisjamison.github.io
+
+Also proud of my week 12 homework that I managed to leverage React + Bootstrap + Express + MongoDB + Heroku for a full stack deployed "app"
```
Comment (rock43053) [inline HIGHLIGHTS.md:15]: Check

Comment (AL743-ni82dy) [inline HIGHLIGHTS.md:15]: Great Job!

File: HIGHLIGHTS.md [hunk 2]
```diff
@@ -7,3 +7,14 @@ What highlights can you come up with to share with the class?
 What do you really like? What can you find for fun facts related to any of these topics online?
 
 ## Cohort Highlights
+I'm very proud of my portfolio template (Although I haven't added the actual data yet)
```
Comment (SBucharelli) [inline HIGHLIGHTS.md:11]: This is really clean looking. Really impressive!

**Review by SBucharelli — APPROVED** [review]

File: HIGHLIGHTS.md [hunk 3]
```diff
@@ -7,3 +7,14 @@ What highlights can you come up with to share with the class?
...
+
+https://hollisjamison.github.io
```
Comment (jessicareynolds) [inline HIGHLIGHTS.md:13]: Nice work!

File: QUESTIONS.md [hunk 4]
```diff
@@ -3,3 +3,15 @@
...
 ## Cohort Questions
+- Do you need to learn multiple frameworks at a high end level or are you ok with just having 1 that you've "mastered" and leave the others in a "familiar" level? (React, Angular, VueJS)
+
+- When should we use server side rendering and when should we use client side rendering?
+
+- How do you feel about people who say you can never learn Javascript until you can make full stack applications without a framework?
+
+- When is the best time to use a relational database vs a non relational (maybe a better question for later weeks)
+
+- How well does Javascript transfer to Python?
```
Comment (jessicareynolds) [inline QUESTIONS.md:22]: Great question! I was curious about this as well.

File: QUESTIONS.md [hunk 5]
```diff
@@ -3,3 +3,15 @@
...
+
+- If there was one technology that this course doesnt have time to teach, and you wish you had more time to cover it, what would that be?
```
Comment (jessicareynolds) [inline QUESTIONS.md:24]: +1

**Pull request merged**